lib/system_registry/task.ex

```diff
@@ -0,0 +1,52 @@
+defmodule SystemRegistry.Task do
+  @moduledoc """
+
+  Creates a process that executes a function anytime the value of a given
```

Comment:
I think this is what you want:
"Creates a process that executes a function anytime the contents of a given system_registry scope change."